test/endpoint-config.test.js

```diff
@@ -216,7 +216,7 @@ test(
       '#define FIXED_PROFILE_IDS { 0x0107, 0x0104, 0x0104 }'
     )
     expect(epc).toContain('#define FIXED_ENDPOINT_TYPES { 0, 1, 2 }')
-    expect(epc).toContain('#define GENERATED_DEFAULTS_COUNT (12)')
+    expect(epc).toContain('#define GENERATED_DEFAULTS_COUNT (19)')
```

Comment:
@tecimovic @brdandu this is interesting

Reply:
That does not make sense. This has to be related to the if condition change.

Reply:
yeah it definitely is, but the condition was completely broken before

Reply:
Hm....

Reply:
Log a bug to thoroughly investigate and validate this change